src/java.base/share/classes/java/io/InvalidObjectException.java

```diff
@@ -48,4 +48,18 @@
     public  InvalidObjectException(String reason) {
         super(reason);
     }
+
+    /**
+     * Constructs an {@code InvalidObjectException} with the given
+     * reason and cause.
+     *
+     * @param reason Detailed message explaining the reason for the failure.
+     * @param cause the cause
+     *
+     * @see ObjectInputValidation
+     * @since 19
+     */
+    public  InvalidObjectException(String reason, Throwable cause) {
+        super(reason);
```

Comment:
Should this be: `super(reason, cause)`?

Reply:
Yes; good catch -- will fix in the next push. Thanks.